pyproject.toml

```
@@ -32,29 +33,29 @@ test = [
     "pytest>=6.0",
     "pytest-xdist>=2.5",
     "pytest-cov>=3.0",
-    "MDAnalysisTests>=2.7.0",
+    "MDAnalysisTests",
```

add MDA for testing ONLY
```suggestion
    "MDAnalysis",
    "MDAnalysisTests",
```